riscv/triggers.cc

```diff
@@ -294,13 +294,14 @@ void mcontrol6_t::tdata1_write(processor_t * const proc, const reg_t val, const
   auto xlen = proc->get_const_xlen();
   assert(get_field(val, CSR_MCONTROL6_TYPE(xlen)) == CSR_TDATA1_TYPE_MCONTROL6);
   dmode = get_field(val, CSR_MCONTROL6_DMODE(xlen));
+  const reg_t maskmax6 = xlen - 1;
```

Comment:
@rtwfroody I didn't find an explicit bound for maskmax6. Does the spec intend ```maskmax6=31``` in RV32 and ```maskmax6=63``` in RV64 for maximum?

Reply:
Yes. It doesn't make sense for maskmax6 to be any larger than that, because then you're just doing an equal comparison.